spark-bigquery-dsv1/spark-bigquery-dsv1-parent/pom.xml

```diff
@@ -39,6 +39,11 @@
       <artifactId>spark-bigquery-dsv1-spark3-support</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
+      <artifactId>commons-lang</artifactId>
```

Comment:
this has been moved according to https://mvnrepository.com/artifact/commons-lang/commons-lang
Please use https://mvnrepository.com/artifact/org.apache.commons/commons-lang3 instead.

Reply:
This was added here because `commons-lang` is used by one of the dependencies of some versions of Spark.
@davidrabinowitz do you remember which one it was?

Reply:
@vishalkarve15 We need `commons-lang` as a provided runtime dependency for OpenLineage